<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WAD Tools</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="baseGrid">
     <item row="0" column="0">
      <widget class="QTabWidget" name="mainTabs">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="mapInfoTab">
        <attribute name="title">
         <string>Map Info</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="mapNamesGb">
             <property name="title">
              <string>Map Names</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QListWidget" name="mapNamesList">
                  <property name="selectionMode">
                   <enum>QAbstractItemView::MultiSelection</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="mapInfoGb">
             <property name="title">
              <string>Map Info</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QListWidget" name="mapInfoList">
                  <property name="selectionMode">
                   <enum>QAbstractItemView::MultiSelection</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="lumpInfoTab">
        <attribute name="title">
         <string>Lump Info</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGroupBox" name="lumpNamesGb">
             <property name="title">
              <string>Lump Names</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="1">
                 <widget class="QListWidget" name="lumpNamesList">
                  <property name="selectionMode">
                   <enum>QAbstractItemView::MultiSelection</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="lumpInfoGb">
             <property name="title">
              <string>Lump Info</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QListWidget" name="lumpInfoList">
                  <property name="selectionMode">
                   <enum>QAbstractItemView::MultiSelection</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="wadPathLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="reload"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="browseWadButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
